README.md

```diff
@@ -73,20 +73,48 @@ Docker images for `pwru` are published at https://hub.docker.com/r/cilium/pwru.
 An example how to run `pwru` with Docker:
 
 ```
-docker run --privileged --rm -t --pid=host -v /sys/kernel/debug/tracing:/sys/kernel/debug/tracing cilium/pwru --filter-dst-ip=1.1.1.1
```

Comment:
`/sys/kernel/debug/tracing` is a standard path for tracing on Linux platform. Have you looked into enabling tracing on your machine?

Reply:
Tracing is enabled on my machine, and I checked that `/sys/kernel/debug/tracing` exists on the host machine. I don't think this path exists on the ubuntu:22.04 container. I can use the `pwru` binary on my machine, as well as use the docker image when I mount `/sys/kernel/debug` instead of `/sys/kernel/debug/tracing`.

Reply:
I can edit the review to add this as a note. Will that be better?

Reply:
>I checked that /sys/kernel/debug/tracing exists on the host machine. I don't think this path exists on the ubuntu:22.04 container.

So I'm not sure where the issue is. The command in the README mounts the tracing directory from the host inside the pwru container. So what do you mean by "I don't think this path exists on the ubuntu:22.04 container"?

Reply:
`/sys/kernel/debug` is RO, and docker cannot create `/sys/kernel/debug/tracing` directory to complete the mount operation for `pwru` container. If the path `/sys/kernel/debug/tracing` existed, then the given command would succeed.  Also, cannot mount this to any other location in the container, because the code checks this path.
```
$ docker run --privileged --rm -t --pid=host -v /sys/kernel/debug/tracing:/sys/kernel/debug cilium/pwru --filter-dst-ip=1.1.1.1
2022/10/11 22:02:43 Failed to retrieve available ftrace functions (is /sys/kernel/debug/tracing mounted?): failed to open: open /sys/kernel/debug/tracing/available_filter_functions: no such file or directory
...
```

Hence, I think it may be worth noting in the README that mounting `/sys/kernel/debug` entirely could act as a workaround.

Reply:
The note looks reasonable to me. Thanks!